infrastructure/code/template/details.html

```diff
@@ -22,6 +22,257 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <style>
+    html {
+      font-size: 16px;
+      /* 1rem = 16px */
+    }
+
+    *,
+    *:before,
+    *:after {
+      box-sizing: border-box;
+    }
+
+    body {
+      height: 100%;
+      width: 100%;
+      margin: 0;
+      padding: 0;
+      overflow: hidden;
+      color: #d4d4d4;
```

Comment:
A question about the colours. Did Design end up using intelliJ theme colours so that when we swap between light/dark and various different themes we can maintain the nice look of the plugin?

Reply:
If it's okay with you, this topic and the scrolling issue you pointed out before will be address when meeting the designer 🏗️